js/buttons.js

```diff
@@ -0,0 +1,103 @@
+function addButtons() {
+  var buttonBox = waterUseViz.elements.buttonBox = svg.append('g')
+    .attr('id', 'button-container');
+  
+  buttonBox.append('rect')
+    .attr('id', 'button-background')
+    .attr('height', waterUseViz.dims.buttonBox.heightDesktop);
+  
+  var buttonY = d3.scaleBand()
+    .domain(['title'].concat(categories))
+    .range([0, waterUseViz.dims.buttonBox.heightDesktop])
+    .paddingInner(0.1);
+  
+  // one group for each button
+  var buttons = buttonBox.selectAll('rect.button')
+    .data(categories)
+    .enter()
+    .append('g')
+    .classed('button', true)
+    .attr('transform', function(d) {
```

Comment:
should we add an `id` for each button group that is the category name?

Reply:
sure, could do that. i don't think we have a direct need for it, but it probably wouldn't hurt